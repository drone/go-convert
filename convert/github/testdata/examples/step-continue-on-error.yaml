# https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error

name: example

jobs:
  job1:
    runs-on: ubuntu-20.04
    steps:
      - name: step1
        run: echo continue on error
        continue-on-error: true
      - name: step2
        run: echo do not continue on error
        continue-on-error: false
      - name: step3                                                           
        run: echo step3                                                        
