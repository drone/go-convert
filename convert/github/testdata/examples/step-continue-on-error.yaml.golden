kind: pipeline
spec:
  stages:
  - name: job1
    spec:
      platform:
        arch: amd64
        os: linux
      runtime:
        spec: {}
        type: cloud
      steps:
      - failure:
          action:
            spec: {}
            type: ignore
          errors:
          - all
        name: step1
        spec:
          run: echo continue on error
        type: script
      - name: step2
        spec:
          run: echo do not continue on error
        type: script
      - name: step3
        spec:
          run: echo step3
        type: script
    type: ci
version: 1
